clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -160,9 +160,35 @@ class ServerGroupController {
   @PostFilter("hasPermission(filterObject?.application, 'APPLICATION', 'READ')")
   @PostAuthorize("@authorizationSupport.filterForAccounts(returnObject)")
   @RequestMapping(value = "/serverGroups", method = RequestMethod.GET)
-  List getServerGroupsByApplications(@RequestParam(value = 'applications') List<String> applications,
-                                     @RequestParam(required = false, value = 'cloudProvider') String cloudProvider) {
-    applications.collectMany { summaryList(it, cloudProvider) }
+  List getServerGroups(@RequestParam(required = false, value = 'applications') List<String> applications,
```

Comment:
Hoping that renaming endpoint methods doesn't cause any problems...

Reply:
Hmm... I feel like `ids` is also misleading, but maybe less so?

Reply:
IMO it's less misleading ... but you're right, hard to say.

Reply:
FWIW for entity tags we did

```
https://api....net/tags?entityId=clouddriver-main-readonly-elb,clouddriver-prestaging-readonly-elb,clouddriver-redis...
```

In this case `entityId` is one of the identity fields on an entity tag. I bring this up as perhaps there's some precedence for going singular `id` vs plural `ids` even if the `id` happens to be comma-separated.

Not sure if we have any other patterns for this in our APIs.

Reply:
Since the other param is `applications` I think `ids` should be used here.  seem ok?

Reply:
👍